test/TestUtilities/Test.Utility/PackageManagement/TestSolutionManager.cs

```diff
@@ -17,19 +17,43 @@ namespace Test.Utility
 {
     public class TestSolutionManager : ISolutionManager, IDisposable
     {
+        private static readonly string GLOBAL_PACKAGES_ENV_KEY = "NUGET_PACKAGES";
+
         public List<NuGetProject> NuGetProjects { get; set; } = new List<NuGetProject>();
+
         public INuGetProjectContext NuGetProjectContext { get; set; } = new TestNuGetProjectContext();
 
+        public string NuGetConfigPath { get; set; }
+
+        public string GlobalPackagesFolder { get; set; }
+
+        public string PackagesFolder { get; set; }
+
         public string SolutionDirectory { get; }
 
-        private const string PackagesFolder = "packages";
 
         private TestDirectory _testDirectory;
 
+        private readonly string _configContent = @"<?xml version='1.0' encoding='utf-8'?>
+<configuration>
+  <packageSources>
+    <add key='NuGet.org' value='https://api.nuget.org/v3/index.json' />
+  </packageSources>
+  <config>
+     <add key='globalPackagesFolder' value='{0}' />
```

Comment:
and should we also overwrite repositoryPath?

Reply:
Don't need to. We set the packages folder when [creating](https://github.com/NuGet/NuGet.Client/blob/581744c1411c484fbe8d68845b5ebb32cb5bbf98/test/TestUtilities/Test.Utility/PackageManagement/TestSolutionManager.cs#L86) a PC project.